connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -120,43 +101,26 @@ public void testInitializeFailure() {
         workerConnector.doShutdown();
         assertStoppedMetric(workerConnector);
 
-        verifyAll();
+        verify(connector).version();
+        verify(offsetStore).start();
+        verify(connector).initialize(any(SourceConnectorContext.class));
+        verify(listener).onFailure(CONNECTOR, exception);
+        verify(listener).onShutdown(CONNECTOR);
+        verify(ctx).close();
+        verify(offsetStorageReader).close();
+        verify(offsetStore).stop();
```

Comment:
I know that this complexity isn't introduced by these changes, and is only translated from the existing EasyMock/PowerMock tests, but it'd be nice if we could try to clean up this logic while we're in the neighborhood.

Would you mind taking a stab at grouping some of these together into reusable methods, like `verifyCleanStart`, `verifyCleanShutdown`, etc.? I tried to do that in https://github.com/apache/kafka/pull/12409 if you're looking for inspiration, but you don't have to match that exact style if you have other ideas/preferences.